controllers/tokenupload_controller.go

```diff
@@ -130,7 +103,8 @@ func (r *TokenUploadReconciler) Reconcile(ctx context.Context, req ctrl.Request)
 		}
 
 		logs.AuditLog(ctx).Info("manual token upload initiated", uploadSecret.Namespace, accessToken.Name)
-		err = r.TokenStorage.Store(ctx, &accessToken, &token)
+		// Upload Token, it will cause update SPIAccessToken State as well
+		err = r.TokenStorage.Store(ctx, accessToken, &token)
 		if err != nil {
 			logError(ctx, uploadSecret, fmt.Errorf("token storing failed: %w", err), r, lg)
 			logs.AuditLog(ctx).Error(err, "manual token upload failed", uploadSecret.Namespace, accessToken.Name)
```

Comment:
This audit log message is IMHO confusing. This is not a manual token upload per se, but rather reading the token data from an annotated secret. IMHO we should distinguish these two cases.

We should make this audit log, because it happened during the access to the 3rd party system (the token storage), but I think we should be able to distinguish between a manual token upload using the REST API and using a Secret in the logs.

Reply:
@metlos 
Agree, but I would say, both plain REST and K8s API (Secret) REST ways are manual (in contrast to OAuth way), no?
So, we probably need to avoid "manual" at all or have something "manual REST" vs "manual K8s"?
WDYT?